### PR TITLE
fix output layout

### PR DESCRIPTION
**Pull request by chrisnojima**
we shouldn't be splitting on `\n`'s etc, else we make a span with nothing inside which isn't correct. this is one text block

### CODE REVIEW

File: shared/crypto/crypto-sub-nav.desktop.tsx [hunk 1]
```diff
@@ -33,11 +33,13 @@ class CryptoSubNav extends React.PureComponent<NavigationViewProps<any>> {
     return (
       <Kb.Box2 direction="horizontal" fullHeight={true} fullWidth={false}>
         <ListAndActiveOperation routeSelected={descriptor.state.routeName}>
-          <SceneView
-            navigation={childNav}
-            component={descriptor.getComponent()}
-            screenProps={this.props.screenProps || noScreenProps}
-          />
+          <Kb.BoxGrow>
```
Comment (chrisnojima) [inline shared/crypto/crypto-sub-nav.desktop.tsx]: else the right side can expand if the contents are large

**Comment by chrisnojima** [comment]
(i didn't test mobile)

**Comment by cjb** [comment]
Works for me.  Thanks!  @thebearjew, okay with you?

**Comment by thebearjew** [comment]
⭐

**Pull request merged**